building/windows.md

```diff
@@ -0,0 +1,240 @@
+# Windows
+
+For now, it is possible to build Phoenix-RTOS images for all targets and run `ia32-generic-qemu` on Windows using
+`Cygwin` or `WSL`. Running the rest of targets is under development.
+
+## Cygwin
+
+To start, we need to download and install the QEMU environment on Windows. It's important to do it first to avoid issues
+with the `qemu-integration` tool.
+

```

Comment:
Let's hide those instructions, it may be obvious for some people.
```suggestion

<details>
<summary> How to install QEMU on Windows (click to expand)</summary>
</details>
```